mythril/analysis/solver.py

```diff
@@ -2,15 +2,13 @@
 from mythril.exceptions import UnsatError
 import logging
 
+
 def get_model(constraints):
-    s = Solver()
-    s.set("timeout", 100000)
 
-    for constraint in constraints:
-        s.add(constraint)
```

Comment:
Now constraints aren't added anymore

Reply:
The constraint class automatically does that, so we may not need it.